src/clients/AcrossConfigStoreClient.ts

```diff
@@ -282,12 +282,15 @@ export class AcrossConfigStoreClient {
 
         // If Token config doesn't contain all expected properties, skip it.
         if (!(rateModelForToken && transferThresholdForToken)) {
-          this.logger.warn({
-            at: "ConfigStore",
-            message: "Ignoring invalid rate model update.",
-            update: args,
-            transferThresholdForToken,
-          });
+          const maxWarnAge = 21_600; // ~3 days @ 12s/block.
```

Comment:
nit: Can you grab the 12s/block hardcoded value from anywhere else so we don't have multiple assumptions in the code?

Reply:
Implemented here: https://github.com/across-protocol/sdk-v2/pull/229/commits/6d6218a891dcffc9a03a151915e4f3f4d112d7d1